etl/command.py

```diff
@@ -322,6 +327,130 @@ def _run_notebook(self) -> None:
                 )
 
 
+class GrapherStep(Step):
+    """
+    A step which ingests data into a local mysql database. You specify it by
+    by making a Python module with a to_grapher_table function that takes a
+    table and returns a sequence of tables with the fixed grapher structure of
+    (year, entitiyId, value)
+    """
+
+    path: str
+    dependencies: List[Step]
+
+    def __init__(self, path: str, dependencies: List[Step]) -> None:
+        self.path = path
+        self.dependencies = dependencies
+
+    def __str__(self) -> str:
+        return f"data://{self.path}"
+
+    def run(self) -> None:
+        # make sure the encosing folder is there
+        self._dest_dir.parent.mkdir(parents=True, exist_ok=True)
+
+        sp = self._search_path
+        if sp.with_suffix(".py").exists() or (sp / "__init__.py").exists():
+            self._run_py()
+
+        else:
+            raise Exception(f"have no idea how to run step: {self.path}")
+
+        # modify the dataset to remember what inputs were used to build it
+        dataset = self._output_dataset
+        dataset.metadata.source_checksum = self.checksum_input()
+        dataset.save()
+
+    def is_dirty(self) -> bool:
+        if not self._dest_dir.is_dir() or any(
+            isinstance(d, DataStep) and not d.has_existing_data()
+            for d in self.dependencies
+        ):
+            return True
+
+        found_source_checksum = catalog.Dataset(
+            self._dest_dir.as_posix()
+        ).metadata.source_checksum
+        exp_source_checksum = self.checksum_input()
+
+        if found_source_checksum != exp_source_checksum:
+            return True
+
+        return False
```

Comment:
```suggestion
    def is_dirty(self) -> bool:
        return True
```

We would need to store checksums in MySQL to know if we need to re-run this step. That will be important to do, but we can defer it until we're happy with the shape of things. Until then, just always say the step needs running.